src/main.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
-import './index.css';
-import './reset.css';
+import './styles/index.css';
+import './styles/reset.css';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
```

Comment:
```suggestion
const rootElement = document.getElementById('root');
if(rootElement === null) throw new Error('Root container missing in index.html')

ReactDOM.createRoot(rootElement).render(
```
![스크린샷 2022-07-11 오후 3 55 25](https://user-images.githubusercontent.com/36851531/178206091-a32ae5e8-5289-4f64-9cad-525145fa601d.png)